### PR TITLE
Added Display Cost field to Accommodation product Rates tab

### PR DESCRIPTION
**Pull request by jessepearson**
#### Changes proposed in this Pull Request:
- Added Display Cost tab to account for offset mentioned in #60 

screen shot 2016-07-07 at 13 59 23

screen shot 2016-07-07 at 14 00 01

-------------------
- [ ] Make sure your changes respect [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- [ ] Did you make changes, or create a **new .js file**? If **Gruntfile.js** exists in the repo, make sure to run `grunt`.

### CODE REVIEW

**Comment by robertstaddon** [comment]
Nice work on this PR, @jessepearson ! I hope they get this merged in soon.

File: includes/admin/class-wc-accommodation-booking-admin-panels.php [hunk 1]
```diff
@@ -47,8 +47,12 @@ public function general_product_data() {
 	 * Loads any CSS or JS necessary for the admin
 	 */
 	public function admin_styles_and_scripts() {
-		$suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
-		wp_enqueue_script( 'wc_accommodation_bookings_writepanel_js', WC_ACCOMMODATION_BOOKINGS_PLUGIN_URL . '/assets/js/writepanel' . $suffix . '.js', array( 'jquery' ), WC_ACCOMMODATION_BOOKINGS_VERSION, true );
+
+		// only load it on products
+		if ( 'product' === get_post_type() ) {
```
Comment (roykho) [inline includes/admin/class-wc-accommodation-booking-admin-panels.php]: Instead of checking for post type, it is best to check the screen id of "product".

File: includes/admin/class-wc-accommodation-booking-admin-panels.php [hunk 2]
```diff
@@ -157,6 +162,10 @@ public function save_product_data( $post_id ) {
 
 			$meta_key = str_replace( '_wc_accommodation_booking_', '_wc_booking_', $meta_key );
 			update_post_meta( $post_id, $meta_key, $value );
+
+			if ( '_wc_booking_display_cost' == $meta_key ) {
```
Comment (roykho) [inline includes/admin/class-wc-accommodation-booking-admin-panels.php]: Why is this needed? It will already be saved without that.

File: includes/admin/views/html-accommodation-booking-rates.php [hunk 3]
```diff
@@ -5,6 +5,13 @@
 			'step' 	=> '0.01'
 		) ) ); ?>
 		<?php do_action( 'woocommerce_accommodation_bookings_after_booking_base_cost', $post_id ); ?>
+
+		<?php woocommerce_wp_text_input( array( 'id' => '_wc_accommodation_booking_display_cost', 'label' => __( 'Display cost', 'woocommerce-accommodation-bookings' ), 'description' => __( 'The cost is displayed to the user on the frontend. Leave blank to have it calculated for you. If a booking has varying costs, this will be prefixed with the word "from:".', 'woocommerce-accommodation-bookings' ), 'value' => get_post_meta( $post_id, '_wc_display_cost', true ), 'type' => 'number', 'desc_tip' => true, 'custom_attributes' => array(
```
Comment (roykho) [inline includes/admin/views/html-accommodation-booking-rates.php]: If the extra conditional check is not needed per my question above, then the key should be `-wc_booking_display_cost`

File: includes/admin/class-wc-accommodation-booking-admin-panels.php (hunk 1 above)
Comment (jessepearson) [inline includes/admin/class-wc-accommodation-booking-admin-panels.php]: Noted and fixing.

File: includes/admin/class-wc-accommodation-booking-admin-panels.php (hunk 2 above)
Comment (jessepearson) [inline includes/admin/class-wc-accommodation-booking-admin-panels.php]: Bookings itself has a field with the id `_wc_display_cost` that is included in the forms that are hidden when an Accommodations product is being worked on. I had to give my field the id `_wc_accommodation_booking_display_cost` in the form, which get changed to `_wc_booking_display_cost` in the loop. I then check for this, and save the value as `_wc_display_cost`, because this is what is actually used to display the price on the front end.

**Comment by jessepearson** [comment]
Closing to create a new PR with fixes.
